x/auth/client/cli/tx_sign.go

```diff
@@ -129,6 +130,10 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 				}
 				// increment the gas
 				newGasLimit += fe.GetTx().GetGas()
+				// Add the fees
+				for _, fee := range fe.GetTx().GetFee() {
```

Comment:
Let's please split out the fees with more descriptive variable for easier comprehension and maintenance:
```go
// Batch transaction fees are unflattened/unmerged
// and need to be combined for the total fee.
// Please see https://github.com/cosmos/cosmos-sdk/issues/18064
unmergedFees := fe.GetTx().GetFee()
for _, fee := range unmergedFees {
   totalFees = totalFees.Add(fee)
}
```